.github/workflows/notebooks_test.yml

```diff
@@ -0,0 +1,47 @@
+name: "Testing Notebooks"
+
+on:
+  push:
+    branches: [master,github-actions-test]
+    paths: 
+    - 'notebooks/**'
+    - 'geomstats/**'
+    - 'tests/**'
+    
+  pull_request:
+    branches: [master]
+    paths: 
+    - 'notebooks/**'
+    - 'geomstats/**'
+    - 'tests/**'
+
+
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6,3.7,3.8]
+        geomstats-backend : ['numpy','pytorch','tensorflow']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build using Python ${{matrix.python-version}} and Backend ${{matrix.geomstats-backend}}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{matrix.python-version}}
+
+      - name: install dependencies [pip]
+        run:  |
+          pip install --upgrade pip setuptools wheel
+          for req in ci-requirements.txt dev-requirements.txt opt-requirements.txt requirements.txt; do
+            pip install -q -r $req
+          done  
+          pip install -e .      
+
+      - name: unit testing [nose2]
+        run: |
+          nose2 --with-coverage tests.notebook_test
```

Comment:
Does code-coverage know how to integrate the code coverage of this workflow with the codecoverage of the build workflow?

Reply:
@ninamiolane which codecov should we upload btw. 3.7,3.8,3.9 ?